cmake/SetupProtobuf.cmake

```diff
@@ -5,10 +5,11 @@ if(LINK_STATIC_LIBPROTOBUF)
     # because `libprotobuf.a` produced by the package manager is not PIC. So we need to
     # build it by ourselves.
 
-    if(UNIX)
-        set(PROTOBUF_DIR ${CMAKE_CURRENT_BINARY_DIR}/protobuf-${PROTOBUF_VERSION})
-        set(PROTOBUF_URL "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.tar.gz")
-        set(PROTOBUF_HASH MD5=f3916ce13b7fcb3072a1fa8cf02b2423)
+    if(UNIX OR MINGW)
+        set(PROTOBUF_VERSION_STATIC "3.6.1")
```

Comment:
I faced [this issue](https://github.com/protocolbuffers/protobuf/issues/4958) when using `3.6.1` in my environment (`3.5.1` seems to work). Could you check that built binaries work in your environment?

Reply:
Hmm, `3.6.1` seems to work in Ubuntu 16.04.

Reply:
Yes, `menoh_test` succeeded on Travis-CI and Appveyor, but I didn't tested it on other environments.

BTW:
If a desirable version of protobuf is changed, I will update PR #82 and rebase this PR on top of it.